### PR TITLE
Support for google_access_context_manager_service_perimeter ingress/egress policies

### PR DESCRIPTION
**Pull request by eadred**
Adds beta support for [egress and ingress policies](https://cloud.google.com/vpc-service-controls/docs/ingress-egress-rules) to VPC Service Control perimeters.

Resolves [Terraform provider Issue 8412](https://github.com/hashicorp/terraform-provider-google/issues/8412).

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
access_context_manager: added `status.egress_policies`, `status.ingress_policies`,
`spec.egress_policies` and `spec.ingress_policies` fields to
`google_access_context_manager_service_perimeter` resource (beta)
```

### CODE REVIEW

**Comment by eadred** [comment]
I'm having trouble running the TF provider tests, but opening this PR in case any review comments come out of it in the meantime.

File: mmv1/products/accesscontextmanager/api.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://accesscontextmanager.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://accesscontextmanager.googleapis.com/v1/
```
Comment (eadred) [inline mmv1/products/accesscontextmanager/api.yaml:23]: Although the new properties are available in the v1 API version, the documentation indicates they are not GA (see the [docs](https://cloud.google.com/vpc-service-controls/docs/ingress-egress-rules)), hence why the same base_url is used for both ga and beta versions.

**Comment by slevenick** [comment]
This looks like a duplicate of https://github.com/GoogleCloudPlatform/magic-modules/pull/4509

**Comment by eadred** [comment]
Closing this as a duplicate of https://github.com/GoogleCloudPlatform/magic-modules/pull/4509.

**Pull request closed without merging**